src/cs-graphics/SetupGLNode.cpp

```diff
@@ -15,6 +16,55 @@ namespace cs::graphics {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
+void GLAPIENTRY MessageCallback(GLenum source, GLenum type, GLuint id, GLenum severity,
```

Comment:
You can use `else if` or early `return` to avoid unneccesary boolean evaluations.